cove_360/lib/threesixtygiving.py

```diff
@@ -439,8 +439,8 @@ class IncompleteRecipientOrg(AdditionalTest):
     """Checks if any grants lack one of either Recipient Org:Postal Code or both of Recipient Org:Location:Geographic Code and Recipient Org:Location:Geographic Code Type"""
 
     check_text = {
-        "heading": "incomplete recipient organisation information",
-        "message": "Your data is missing either Recipient Org: Postal Code or Recipient Org: Location:Geographic Code combined with Recipient Org: Location: Geographic Code Type. Knowing the geographic location of recipient organisations allows users of your data to understand your data and combine it with other data sets to form a broader picture of grant-making."
+        "heading": "not have recipient organisation location information",
```

Comment:
@BobHarper1 you'll need to change the verb in order for this to make sense. See https://github.com/OpenDataServices/cove/blob/master/cove_360/lib/threesixtygiving.py#L434 for an example .

Reply:
verb added "do"